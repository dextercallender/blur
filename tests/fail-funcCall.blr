int add(int x, int y) {
	int z = x + y;
	return z;
}

int sub(int x, int y) {
	int z = x-y;
	return z;
}
int mult(int x, int y) {
	int z = x*y;
	return z;
}
int main() {
	add(5, 4);
	sub(3,2);
	minus(5,4);
	return 8;
}