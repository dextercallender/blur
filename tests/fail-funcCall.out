Fatal error: exception Failure("undeclared function minus")
